### PR TITLE
feat(ST-51): add material form modal

### PR DESCRIPTION
**Pull request by akib-raihan-sazim**
## Description of Change
- Updated the ClassroomNav, added classwork component
- Created ClassworkContainer component
- Created CreateFileButton, MaterialFormModal
- Created separate file for styles and interface files

## Associated Ticket Link(s)

- https://trello.com/c/Sewe3ouV

## Screenshots / Screen Recordings
N/A

### CODE REVIEW

File: modules/Classroom/components/MaterialFormModal/MaterialFormModal.tsx
Comment (ishmam-iqbal-sazim) [inline modules/Classroom/components/MaterialFormModal/MaterialFormModal.tsx:1]: Let's be more precise with the names. You can name it something like CreateMaterialFormModal. Same applies for the hooks, component names and types

File: modules/Classroom/components/MaterialFormModal/MaterialFormModal.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+  const onSubmit = (data: TMaterialFormValues) => {
+    // TODO : Add API call here
+    setIsLoading(true);
+    console.log(data);
+    setTimeout(() => {
+      setIsLoading(false);
+      reset();
+      onClose();
+    }, 1000);
```
Comment (ishmam-iqbal-sazim) [inline modules/Classroom/components/MaterialFormModal/MaterialFormModal.tsx:43]: Is this timeout just a placeholder?

Comment (akib-raihan-sazim) [reply modules/Classroom/components/MaterialFormModal/MaterialFormModal.tsx:43]: yes bhaiya.

**Review by ragib-hossain-sazim — APPROVED** [review]
Approved pending comment

**Pull request merged**